apiary.apib

```diff
@@ -1688,6 +1688,103 @@ For example you should create a group for the Introduction to Programming course
             }
         }
 
+## Group [/v1/issuer/groups/{group_id}/clone]
+
+### Clone a Group [POST]
+You can clone a group using this action. Add attributes to override cloned values. Cloning a group only works when certificate or badge designs are within the same department as the group being cloned.
```

Comment:
Can you clarify why you mean here. Do you want to also mention what's here in the department  attribute description?

Reply:
I think this EP doesn't need to support `department_id` in the request body since it always has to be the same as the original group. If we remove `department_id` from the request body, the current wording here should be fine.